files/etc/radios.json

```diff
@@ -519,6 +519,15 @@
       "beamwidth": 7
     }
   },
+  "mikrotik DiscG 5 ac xl (rbdiscg-5acd)": {
```

Comment:
Lower case here.

Reply:
I think I edited it right. I'm just refamiliarizing myself with the web editor.